crates/viewer/re_view_spatial/src/visualizers/encoded_depth_image.rs

```diff
@@ -0,0 +1,178 @@
+                let all_blobs_indexed = iter_slices::<&[u8]>(&all_blob_chunks, timeline);
+                let all_formats_indexed =
+                    iter_component::<ImageFormat>(&all_format_chunks, timeline);
+                let all_media_types = results.iter_as(
+                    timeline,
+                    EncodedDepthImage::descriptor_media_type().component,
+                );
+                let all_colormaps =
+                    results.iter_as(timeline, EncodedDepthImage::descriptor_colormap().component);
+                let all_value_ranges = results.iter_as(
+                    timeline,
+                    EncodedDepthImage::descriptor_depth_range().component,
+                );
+                let all_depth_meters =
+                    results.iter_as(timeline, EncodedDepthImage::descriptor_meter().component);
+                let all_fill_ratios = results.iter_as(
+                    timeline,
+                    EncodedDepthImage::descriptor_point_fill_ratio().component,
+                );
+
+                let entity_path = ctx.target_entity_path;
+
+                let data_iter = re_query::range_zip_1x6(
+                    all_blobs_indexed,
+                    all_formats_indexed,
+                    all_media_types.slice::<String>(),
+                    all_colormaps.slice::<u8>(),
+                    all_value_ranges.slice::<[f64; 2]>(),
+                    all_depth_meters.slice::<f32>(),
+                    all_fill_ratios.slice::<f32>(),
+                )
+                .filter_map(
+                    move |(
+                        (_time, row_id),
+                        blobs,
+                        format,
+                        media_type,
+                        colormap,
+                        value_range,
+                        depth_meter,
+                        fill_ratio,
+                    )| {
+                        let blob = blobs.first()?;
+                        let format = first_copied(format.as_deref())?;
+                        let media_type = media_type
+                            .and_then(|types| types.first().cloned())
+                            .map(|mt| MediaType(mt.into()));
+
+                        let image = match ctx.store_ctx().caches.entry(
+                            |c: &mut ImageDecodeCache| {
+                                c.entry_encoded_depth(
+                                    row_id,
+                                    EncodedDepthImage::descriptor_blob().component,
+                                    blob,
+                                    media_type.as_ref(),
+                                    &format,
+                                )
+                            },
+                        ) {
+                            Ok(image) => image,
+                            Err(err) => {
+                                re_log::warn_once!("Failed to decode EncodedDepthImage at path {entity_path}: {err}");
```

Comment:
didn't exist when this PR started, but this should be a visualizer error now that goes on the output :)